.github/workflows/docker.yml

```diff
@@ -24,7 +24,7 @@ permissions:
 
 jobs:
   docker:
-    name: Docker (${{ matrix.image }} / ${{ matrix.network }}) [publish=${{ (inputs.ref || github.ref) == 'refs/heads/master' || startsWith(inputs.ref || github.ref, 'refs/tags/') }}]
+    name: Docker (${{ matrix.image }} / ${{ matrix.network }}) [publish=${{ github.event_name != 'pull_request' }}]
```

Comment:
what's the implication of this change? do we get docker builds for all branches now?

Reply:
What's new is building docker images on pull requests to master and release branches (they are not published anywhere though). All the publishing remains as it was before.